prime-dotnet-webapi/Controllers/SitesController.cs

```diff
@@ -803,6 +803,29 @@ public async Task<ActionResult<Site>> DeclineSite(int siteId)
             return Ok(ApiResponse.Result(updatedSite));
         }
 
+        // PUT: api/Sites/5/enable-editing
+        /// <summary>
+        /// Enable editing a site
+        /// </summary>
+        /// <param name="siteId"></param>
+        [HttpPut("{siteId}/enable-editing", Name = nameof(EnableEditingSite))]
+        [Authorize(Roles = Roles.EditSite)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
+        [ProducesResponseType(StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
Now that you have updated this to 200OK from 204NoContent, this should have the return type of the endpoint:
```C#
[ProducesResponseType(typeof(ApiResultResponse<Site>), StatusCodes.Status200OK)]
```

Reply:
fixed in 53ba7fd